docs/src/main/tut/chisel3/reset.md

```diff
@@ -3,6 +3,21 @@ layout: docs
 title:  "Reset"
 section: "chisel3"
 ---
+
+```scala mdoc:invisible
+import chisel3._
+import chisel3.stage.ChiselGeneratorAnnotation
+
+class Submodule extends MultiIOModule
+
+trait NewIn3p3 {
+  throw new Exception("This is a new feature coming in 3.3!")
+}
+
+trait RequireSyncReset extends NewIn3p3
+trait RequireAsyncReset extends NewIn3p3
+```
+
```

Comment:
What does this look like after running mdoc?

I really need to get this running so I can generate it locally

Reply:
In my head I was just going to put

_Coming in Chisel 3.3_ above the description of it

Reply:
This whole block isn't rendered at all. We don't compile so the exception never happens. This kludges around letting us define `RequireSyncReset` and use it, but then added feedback that this will no longer compile (due to a name collision) once we bump to 3.3.

The safest thing is to chunk the PR into two PRs: (1) a first PR that doesn't include the 3.3 features and (2) a second PR that adds that section.

Reply:
Oh that's clever

Sure, I can split it into 2 PRs, I'll merge your changes then chop out the 3.3 stuff